fastavro/_logical_writers.pyx

```diff
@@ -0,0 +1,234 @@
+# cython: language_level=3str
+
+"""Python code for writing AVRO files"""
```

Comment:
I would remove this line and the comment lines below it. I don't think this line makes sense in this new file and I don't think the comment lines are accurate for this file either (as we had implemented logical types before the apache repo)